src/site/content/en/blog/link-prefetch/index.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Prefetch resources to speed up future navigations
+subhead: |
+    Learn about rel=prefetch resource hint and how to use it.
```

Comment:
Some comments:

- The subhead doesn't accept the code formatting, so it will appear as plain <link rel="prefetch">.
- The description hides anything between <>, so I just left: rel="prefetch".

Reply:
If we use that in the subhead or the description, it will hide whatever is inside the < >.
Just left it as it was.